examples/react-native/__tests__/ts/realm-database/configure-realm-sync.test.tsx

```diff
@@ -30,7 +30,80 @@ function AppWrapperSync() {
   return (
     <AppProvider id={APP_ID}>
       <UserProvider>
-        <RealmProvider sync={{flexible: true, onError: error => console.error(error)}}>
+        <RealmProvider
+          sync={{
+            flexible: true,
+            onError: error => console.error(error),
```

Comment:
idk this change would be unclear, but you could shorten this to:

```suggestion
            onError: console.error,
```